### PR TITLE
Homepage todo

### PR DESCRIPTION
**Pull request by Mohitk391**
Added Homepage and Todo page

Files to check: 
1) Homepage.js
2) style.css
3) Todo.js
4) Todo.css
5) TaskContext.js
6) TaskReducer.js

preview: https://62da853bb3e78d064b9c492a--magnificent-sherbet-942505.netlify.app/

### CODE REVIEW

File: src/pages/todo/Todo.css [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+.task-contents input, #taskDescription{
+    border: none;
+    background-color: var(--bg-secondary);
+    outline:none;
+    border-radius: 10px;
+    padding: 5px 10px;
+}
+
+#taskDescription{
+    height: 100px;
```
Comment (abhisheksingh98) [inline src/pages/todo/Todo.css:107]: use rem instead of px for dimensions greater than 10px

File: src/pages/todo/Todo.css [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    resize: none;
+}
+
+.time-input{
+    justify-content: space-evenly;
+}
+
+.time-input input{
+    width:70px;
```
Comment (abhisheksingh98) [inline src/pages/todo/Todo.css:116]: use rem instead of px for dimensions greater than 10px

**Review by abhisheksingh98 — COMMENTED** [review]
fix the dimension units, rest looks good and clean to me

**Comment by varun-Tad** [comment]
Looks good to me.You can go ahead and merge

**Pull request merged**